framework/caching-cmm.md

```diff
@@ -5,15 +5,24 @@
 
 ## Version
 
-0.1.0-preview
+0.3.0
+
+### Changelog
+
+- 0.3.0
```

Comment:
What happened to 0.2.0?

Reply:
This PR is intended to go in after #176 , which represents 0.2.0, but I didn't want to stack this PR on top of another when the base PR might change.

Reply:
I get it.  Ideally we would have a better way to handle this.

Reply:
IMO, the version should change in whatever order the merges happen.